tests/react-directive-spec.js

```diff
@@ -33,6 +33,17 @@ var Hello = React.createClass({
   }
 });
 
+var Transcluder = React.createClass({
+  propTypes: {
+    children : React.PropTypes.string,
+  },
+
+  render() {
+    var {children} = this.props;
+    return <div dangerouslySetInnerHTML={ {__html: children} }/>;
```

Comment:
is it possible to set the `dangerouslySetInnerHTML` in the `renderComponent` function (https://github.com/ngReact/ngReact/blob/master/ngReact.js#L124-L128) so someone doesn't have to set it directly in the React class? You could pass in a flag if the transclude setting has been set.